CorsixTH/Lua/utility.lua

```diff
@@ -18,6 +18,12 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE. --]]
 
+if _G._VERSION == "Lua 5.2" or _G._VERSION == "Lua 5.3" then
```

Comment:
I think we don't use this function anymore, or shouldn't.

Reply:
It's still used here in "config_finder.h", at line 808:

https://github.com/leiget/CorsixTH/blob/9cdf151af7a5e4e0d995a0cdac1802486efc4d1b/CorsixTH/Lua/config_finder.lua#L807

If we should use a different function, what one would it be?

Reply:
Nevermind, I just saw the second comment after this. Changed.